index.html

```diff
@@ -108,7 +108,7 @@
             Firecracker can run Linux and
             <a href="http://blog.osv.io/blog/2019/04/19/making-OSv-run-on-firecraker">OSv</a>
             guests. Our latest roadmap can be found
-            <a href="https://github.com/firecracker-microvm/firecracker/projects/13">here</a>.
+            <a href="https://github.com/firecracker-microvm/firecracker/projects/42">here</a>.
```

Comment:
I'm getting 404 on this link as well. The link that works for me is https://github.com/orgs/firecracker-microvm/projects/42
```suggestion
            <a href="https://github.com/orgs/firecracker-microvm/projects/42">here</a>.
```

Reply:
Oops! Good catch!

Reply:
Fixed!